src/pages/composites/categories/index.tsx

```diff
@@ -24,8 +30,7 @@ const composite = () => {
 				BGColor="white"
 				style={{ display: 'flex', flexDirection: 'column' }}
 				header={{ title: 'All Categories', align: 'left' }}>
-				<Categories categories={categories} />
-				<ViewMoreLink href="/composites/products" text="View more Products" />
```

Comment:
I personally think this could be reused because it has taken care of the icon. But you can delete it if you don't want to use it again

Reply:
I think you may be looking at an older version of the file. I can't see what you are referring to

Reply:
I'm talking about line 28